code/src/Urls.js

```diff
@@ -0,0 +1 @@
+export const ThoughtsUrl = 'https://happy-thoughts-technigo.herokuapp.com/thoughts';
```

Comment:
Since url is not a component, we should rather name it in lowercase :) This would be correct way :

```
export const thoughtsUrl = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'; 
```